.travis.yml

```diff
@@ -7,10 +7,13 @@ cache:
   directories:
     - $HOME/.pip-cache/
     - $HOME/virtualenv/python3.5
-script:
-  # Travis will automatically detect requirements.txt and run pip install
+install:
+  - pip install -r requirements.txt
```

Comment:
It is; if you override install at all, Travis no longer defaults to `pip install -r ...`